Expose options_per_vote setting and add permanent_gamba_option setting

inks007 opened the pull request:
Allow options_per_vote to be set via the webUI.
Add an option where one vote option will always be gamba.
Update manifest.json.

inks007 committed:
Expose options_per_vote setting and permanent_gamba_option setting.
Update manifest.json.

Miczu commented:
I'm not against having this as an additional option, but I wanted to confirm who is the target audience for this.

inks007 commented:
The options_per_vote setting might prove useful in balancing. Now that we have so many more outcomes, that the chances of getting effective counters in vote options are decreasing. Decreasing the vote duration to compensate tends to lead to more chaos, whereas increasing the number of options is more tactical. If you leave that at 4, then it's business as usual.

This might also make long vote duration TI runs more feasible, since you are less likely to be stuck on a set of bad options.

The setting to replace one option permanently as gamba was just for a little bit of fun. Also might be good for long duration TI. The default is still business as usual.

Miczu commented:
Can you move the new inputs/options lower? Time range changes frequently and having it at the top is convenient. Changing the number of vote options won't change as much and can be lower.

Miczu commented:
![image](https://github.com/user-attachments/assets/16c36000-08cb-47b0-abbc-e7d6042d0c4d)
Looks like the basics are covered. 
Gamba works and appears at the end. 
From gameplay perspective all good 👍

inks007 committed:
Preserve outcomes.gamba object
Don't edit the outcomes.gamba settings object - outcomes.gamba.enabled now follows settings properly when disabling permanent_gamba_option.
Reordered the noita settings page.

Miczu reviewed (approved):
Looks good to me

Miczu commented:
Thanks for the PR. Chase and chat was having fun by messing with the extra vote options. Perma gamba is like meth for chat and Chase is limiting it.
